server/app/controllers/DocumentCtrl.js

```diff
@@ -0,0 +1,238 @@
+import db from '../models/index';
+import * as auth from '../helpers/AuthHelper';
+
+/**
+ * createDocument
+ * Create a new document
+ * @param {Object} req The Request object
+ * @param {Object} res The Response from the server
+ * @return {undefined}
+ */
+const createDocument = (req, res) => {
+  db.Documents
+    .create({
+      title: req.body.title,
+      content: req.body.content,
+      owner: req.user.userId,
+      access: req.body.access || 'public'
+    })
+    .then(document => res.status(201).json({ message: 'Document Created', document }))
+    .catch(error => res.status(500).json({ message: 'Error Creating Document', error }));
+};
+
+/**
+ * getDocuments
+ * Fetch and return all documents in the database
+ * @param {Object} req The Request object
+ * @param {Object} res The Response from the server
+ * @return {undefined}
+ */
+const getDocuments = (req, res) => {
+  if (!auth.userIsAdmin(req.user.role)) {
+    return res.status(403).send({
```

Comment:
I think http status code 401 `Unauthorized` is more appropriate here rather than `403: Forbidden`